### PR TITLE
Add `show_envvar` to display env variables in help

### LINKED ISSUE #6173: Add documentation for PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP
The ability for regular users to create organizations within a Parsec Server instance is configured via the `PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP` variable during deployment. 

This is not well documented in [Parsec Server configuration](https://docs.parsec.cloud/en/latest/adminguide/hosting/index.html#parsec-server-configuration) and it is particularly important for self-hosted deployments.

```
# Allow bootstrapping organizations without prior creation.
PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP=false
```
We should add some more detail to the variable description: briefly explain what does it means to bootstrap an organization (link to the relevant section in the docs?), which users/profiles are allowed to do it, what is the default behavior (if variable is not defined), etc..

Also, maybe add a Warning notice suggesting to set the value to `false` for on-premise deployments.

### PR DESCRIPTION
**Pull request by mmmarcos**
Also, update Administrator Guide:
- Remove `PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP` from docs since it has the expected default value (False)
- Add note explaining how to get the full list of available env var

Closes #6173

### CODE REVIEW

**Review by FirelightFlagboy — APPROVED** [review]
LGTM

File: docs/adminguide/hosting/parsec.env [hunk 1]
```diff
@@ -3,27 +3,25 @@ PARSEC_HOST=0.0.0.0
...
+
 # The log file to write to (default to `stderr`).
 # PARSEC_LOG_FILE
 
-# The URL to reach Parsec server
+# The URL to reach Parsec server.
 PARSEC_BACKEND_ADDR=parsec://127.0.0.1:6777
 
-# Allow organization bootstrap without prior creation.
-PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP=false
```
Comment (vxgmichel) [inline docs/adminguide/hosting/parsec.env:22]: I think it's still useful to have those variables documented in the template.

Maybe:
```
# Uncomment to allow organization bootstrap without prior creation.
# PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP=true
```

**Review by vxgmichel — APPROVED** [review]
One comment, LGTM otherwise

File: docs/adminguide/hosting/parsec.env (hunk 1 above)
Comment (mmmarcos) [reply docs/adminguide/hosting/parsec.env:22]: I agree with you. For the moment, I will merge this PR as is and will open another issue to document all variables in this file.

Comment (mmmarcos) [reply docs/adminguide/hosting/parsec.env:22]: Done #6198

**Pull request merged**